frontend/src/ui/Results/Results.stories.tsx

```diff
@@ -5,27 +5,44 @@ import occupations from '../../testing/data/occupations';
 import states from '../../testing/data/states';
 import transitionData from '../../testing/data/transitionData';
 import Results, { ResultsProps } from './Results';
+import { Provider } from 'react-redux';
+import { createStore } from '../../ducks';
 
 export default {
   title: 'JobHopper/Results',
   component: Results,
 } as Meta;
 
-const Template: Story<ResultsProps> = args => <Results {...args} />;
+const Template: Story<ResultsProps> = args => (
+  <Provider store={createStore()}>
```

Comment:
Rather than provide the Results component with a redux store, I think we want to remove the redux usage from the Result component. Redux is anything from src/ducks. So it looks like we need to remove this line https://github.com/codeforboston/jobhopper/blob/d73ad6030c6a8fe8866c95fd3de5b1915e81ef9a/frontend/src/ui/Results/Results.tsx#L37 from the component. The component already has selectedState as a prop so I believe we can replace usages of state in there with selectedState

Reply:
fixed.